kubernetes/config-maps/nginx/ssl.conf

```diff
@@ -0,0 +1,26 @@
+# Disable SSLv3
+ssl_protocols              TLSv1 TLSv1.1 TLSv1.2;
+
+# Enable server-side protection
+ssl_prefer_server_ciphers  on;
+ssl_ciphers                "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
+
+# Specify curve P-384
+ssl_ecdh_curve             secp384r1;
+
+# Improve ssl performence
+ssl_session_cache          shared:SSL:10m;
+ssl_session_timeout        5m;
+
+# Enable OCSP stapling
+ssl_stapling               on;
+ssl_stapling_verify        on;
```

Comment:
Please check what happens when OCSP stapling is enabled while using a self-signed certificate. I.e. whether nginx tries to contact the stapling responder and fails.

Stapling makes sense only with certificates singed by a CA because the whole feature is just that nginx asks the CA if the certificate is still valid. If there's no CA (the certificate is self-signed), it obviously can't be done. We want it if we can have it but with self-signed certificates we can't so it's best if nginx can just ignore in that case. But if it can't, we need a variable that enables/disables stapling.

Reply:
Ok, so I deleted this settings from config.